src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: normal;
+}
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your
web document